src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1174,6 +1177,8 @@ def getWorkloadAssignArgs():
                      # set to "" string or None for eos-lfn-prefix if you don't want to save the log in eos
                      "Override": {"default": {"eos-lfn-prefix": "root://eoscms.cern.ch//eos/cms/store/logs/prod/recent/PRODUCTION"},
                                   "type": dict},
+                     # Rucio rule subscription lifetime (used in ContainerRules by T0)
+                     "DatasetLifetime": {"default": 18*30*24*60*60, "type": int, "assign_optional": True},
```

Comment:
same comment here, regarding the default value. The best would be to set it to 0 IMO.
I would suggest to define a `validate` function as well, probably something like `lambda x: x >= 0`.

Reply:
Sure. looks reasonable